debvader/deblender.py

```diff
@@ -291,6 +304,37 @@ def get_deblending_meta_data(self, res_deblend=None):
 
         return res_deblend_meta
 
+    def calc_epistemic(self, cutout_image, output_image_mean):
+        """
+        Compute epistemic uncertainty (from the decoder of the deblender)
+
+        parameters:
+            cutout_image: image for which epistemic uncertainty is to be determined
+            output_image_mean: mean of the output distribution predicted for each pixel
+        """
+        if self.epistemic_uncertainty_estimation:
+            epistemic_uncertainty = deblend(
+                np.std(
+                    deblend(
+                        self.net,
+                        np.array([cutout_image] * 100),
+                        normalizer=self.normalizer,
+                    )[0],
+                    axis=0,
+                )
+            )
```

Comment:
Could you add a little comment in the docstring about what computation is performed ?
I'm a bit confused by the `deblend(std(deblend(...)))`

My guess is that this is a mistake and the first `deblend` is actually a `list`.

Reply:
I agree. This is somewhere I wanted to ask Bastien what he was trying to do
In my opinion... He was probably just trying to feed cutouts to the deblender 100 times to see the epistemic uncertainty but I am not sure

Reply:
I think just removing the first deblend would solve the issue here. 
Then np.std is calculating the pixel-wise std over the predictions.

I just push a commit (after testing that the function works)